### ISSUE TITLE
[BUG] Policy violations do not update when another component ver is selected

### ISSUE THREAD

**Issue by f5oto** [sonatype-nexus-community/vscode-iq-plugin#232]
**Describe the bug**
Policy violations do not update when you select a different component version

**To Reproduce**
Steps to reproduce the behaviour:
1. Select a component to view the violations
2. Navigate to "Policy" tab
3. Select a different version of the component
4. The CIP doesn't update with that versions policy violations.

**Expected behaviour**
Same as the other tables (component info / Security and Licensing), the policy violations should be looked up and updated

**Screenshots**
![image](https://user-images.githubusercontent.com/51363621/157492769-c1315b1f-a3f3-4650-aa74-6caf1c7b156a.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - NodeJS Version: 14.16.0
 - VS Code Version: 1.65.0
 - Version of Extension: 1.2.3

**Additional context**

cc @bhamail / @DarthHater

**Comment by madpah** [comment]
Thanks for the issue @f5oto.

tldr: The extension is working as we'd expect as the Policy Violations obtained from Nexus Lifecycle only relate to the version currently detected in the project.

I'll convert this to an Enhancement :-)
